content/cpp/concepts/inheritance/terms/multiple-inheritance/multiple-inheritance.md

```diff
@@ -0,0 +1,133 @@
+---
+Title: 'Multiple Inheritance'
+Description: 'Enables classes to inherit characteristics from more than one class.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Abstraction'
+  - 'Classes'
+  - 'Objects'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+**Multiple inheritance** is a type of [inheritance](https://www.codecademy.com/resources/docs/cpp/inheritance) where classes can inherit from more than one class.
+
+## Syntax
+The syntax for multiple inheritance is similar to the syntax for [single inheritance](https://www.codecademy.com/resources/docs/cpp/inheritance).
+```cpp
+class DerivedClass : public BaseClass1, public BaseClass2 {
+    // Derived class members
+};
+```
+Classes can inherit from as many base classes as necessary and, as with single inheritance, access modifiers can be used to modify access to inherited members.
+
+The ordering of base classes affects the ordering of calls to [constructors](https://www.codecademy.com/resources/docs/cpp/constructors) and [destructors](https://www.codecademy.com/resources/docs/cpp/destructors). Virtual base class constructors are called first, from left to right, followed by direct base class constructors from left to right. Destructors are called in reverse order to constructors.
+
+## Examples
+
+### Basic multiple inheritance
+```cpp
+class Bat { 
+  public: 
+    void fly();
+}; 
+
+class Man { 
+  public: 
+      void run();
+};
+
+class ManBat : public Man, public Bat { 
+  // run() and fly() are both available as they're inherited from Man and Bat 
+}; 
+```
```

Comment:
Please add at least one new line before starting a new section; helps in readability while editing.